src/server/endpoint.cc

```diff
@@ -88,7 +88,7 @@ namespace Pistache::Http
             if (entry.getTag() == Polling::Tag(timerFd))
             {
                 uint64_t wakeups;
-                ::read(timerFd, &wakeups, sizeof wakeups);
+                [[maybe_unused]] auto rv = ::read(timerFd, &wakeups, sizeof wakeups);
```

Comment:
Why? If you want to ignore the result value, just cast the result to `void`

Reply:
The compiler on one of our build systems (I don't remember whether it was a recent version of g++ or clang) was not happy about the cast.  The only way I could find to avoid the warning was to add the attribute.